Button/Button.html

```diff
@@ -1,4 +1,4 @@
-<template attach-point="focusNode">
+<template attach-point="focusNode" aria-label="{{!this.showLabel ? this.label : ''}}">
```

Comment:
Actually, thinking about this more, I don't think setting `aria-label` to `""` is safe, because `aria-label` overrides the inner text of the `<button>`.  So you can end up with a button like `<button aria-label="">Submit</button>` and then a screen reader may announce that the `<button>` has no label (i.e. a blank label).

In other words `<button aria-label="">Submit</button>` is different than `<button>Submit</button>`.